src/legacy/core_plugins/timelion/server/series_functions/__tests__/yaxis.js

```diff
@@ -95,16 +95,16 @@ describe('yaxis.js', () => {
 
   it('throws an error if currency is not three letter code', () => {
     invoke(fn, [seriesList, 1, null, null, null, null, null, 'currency:abcde']).catch(e => {
-      expect(e).to.be.an(Error);
+      expect(e).to.be.an.instanceof(Error);
```

Comment:
These tests were actually not running correctly and just silently failed without breaking the test itself. It triggered an exception that was caught by the outer `catch` function which then logged an "unhandled promise rejection" desperation warning, which in turn went unnoticed.